### ISSUE TITLE
Why doesn't scale work for area

### ISSUE THREAD

**Issue by devcrocod** [Kotlin/kandy#236]
## **Describe the bug**
Please provide a description of the bug, including steps to reproduce it and the expected result.

## **To Reproduce**
```kts
plot {
            area {
                x(listOf("January", "February", "March", "April", "May")) {
                    axis.name = "months"
                }
                y(listOf(200, 150, 300, 250, 420)) {
                    axis.name = "sales"
                    scale = continuous(min = 100, max = 500)
                }

                alpha = 0.5

                fillColor = Color.BLUE

                borderLine {
                    color = Color.BLACK
                    width = 1.5
                }
            }
        }
```

## **Screenshots**
![areaTest](https://github.com/Kotlin/kandy/assets/19648120/f993b05e-6d28-4124-ad6f-07ca41efaa9f)

**Comment by AndreiKingsley** [comment]
It's a lets-plot behavior. Same for bars. I consider that's bug.

**Comment by AndreiKingsley** [comment]
https://github.com/JetBrains/lets-plot-kotlin/issues/219

**Comment by AndreiKingsley** [comment]
Should be fixed via #358
